package.json

```diff
@@ -18,7 +18,7 @@
     "start": "pnpm build && node ./dist/worker/worker.js",
     "prepare": "husky",
     "ci_validate": "node ./dist/bin/validateJSONRecipients.js",
-    "validate": "pnpm tsc && node ./dist/bin/validateJSONRecipients.js"
+    "validate": "pnpm build && node ./dist/bin/validateJSONRecipients.js"
```

Comment:
This was incorrectly running commit checks on the _last built artifact_ instead of the current code in the repo.